web/src/components/ListItem.tsx

```diff
@@ -34,6 +34,10 @@ const Title = styled.div`
   font-weight: 700;
 `
 
+const FullWidthLink = styled(StyledCleanLink)`
```

Comment:
Why are you not using the default export component?

Reply:
because we can use the standard styles from the styled clean link. If not i would have to duplicate the style code

Reply:
Yeah I got that but why the named export `StyledCleanLink` and not the deafult export `CleanLink` which internally uses the `StyledCleanLink`?